lib/jazzy/themes/apple/assets/js/jazzy.js

```diff
@@ -57,3 +57,17 @@ $("a:not('.token')").on('click', function() {
     openCurrentItemIfClosed();
   }
 });
+
+// KaTeX rendering
+const smallBlockRegex = /^\$.*\$$/gm; // Check for $...$ in code blocks
+const bigBlockRegex = /\$\$(\n|.)*\$\$/gm // Check for $$...$$ in code blocks
+document.querySelectorAll('code').forEach((block) => {
+                const content = block.innerHTML
+               if (smallBlockRegex.test(content)) {
+                            const eq = content.slice(1, -1)
+                            katex.render(eq, block)
+               } else if (bigBlockRegex.test(content)) {
+                            const eq = content.slice(2, -2)
+                            katex.render(eq, block)
+               }
+});
```

Comment:
Regexps: `g` flag is not required and will confuse with its `lastIndex` behaviour.  Look at the `s` flag for the `\n|.`pattern?  Missing `^$` anchors in the `bigBlockRegex` pattern?

Needs to not match in `<pre><code>` elements -- that is supposed to be verbatim.

Should test the `bigBlockRegex` first otherwise small will match (the accidental `g` flag sometimes saves it!)

Might be better to write a single regexp and use a capture group to pull out the equation to render: what's here gets confused by trailing newlines.

If katex fails then it would be good to fall back to what the user wrote - when I tried I just got blank content.

Style: looks to me as though this file uses semicolons, 2-space indent, and jQuery?